content/en/load-testing/Advanced/010-merge-rules.md

```diff
@@ -31,8 +31,8 @@ statusCodePattern [s] .... reg-ex defining a matching status code
 urlPattern [u] ........... reg-ex defining a matching request URL
 runTimeRanges [r] ........ list of run time segment boundaries
 
-stopOnMatch .............. whether or not to process the next rule even if
-                           the current rule applied (defaults to true)
+stopOnMatch .............. whether or not to process the next rule even if the current rule applied (defaults to true)
+dropOnMatch .............. whether or not to discard a matching request instead of renaming it (defaults to false). If the rule applies, the request processing will stop.
```

Comment:
That should not be here because these are my changes which are in the master. Something went wrong. You PR should only contain why you did vs. the master.